mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping

```diff
@@ -62,3 +73,6 @@ CLASS net/minecraft/class_2794 net/minecraft/world/gen/chunk/ChunkGenerator
 		ARG 2 z
 	METHOD method_27997 withSeed (J)Lnet/minecraft/class_2794;
 		ARG 1 seed
+	METHOD method_28506 getCodec ()Lcom/mojang/serialization/Codec;
+	METHOD method_28507 isStronghold (Lnet/minecraft/class_1923;)Z
```

Comment:
Is this defining whether this chunk generator can populate strongholds?

Reply:
It takes a ChunkPos to query whether a stronghold should generate in that chunk. Not sure what the best name would be for it. Maybe `containsStronghold`?

Reply:
`isStrongholdAt`?